### PR TITLE
MAUI - 874348 : Added theme keys for MAUI Image editor

### PR DESCRIPTION
**Pull request by Shalini-Ashokan**
### Description
Added theme keys for MAUI image editor control.

### CODE REVIEW

File: MAUI/Themes/Keys.md [hunk 1]
```diff
@@ -107,3 +107,1809 @@ This page lists the keys for each control and the element to which it is mapped
...
+           SfImageEditorTheme 
+            <br/>
+            <br/>
+        </td>
+        <td>    
+            By merging this key in application resources, it is possible to customize the appearance of the SfImageEditor without merging common theme resource and control style resource dictionaries.
+			
+{% highlight xaml %}
+
+<Application xmlns:syncTheme="clr-namespace:Syncfusion.XForms.Themes;assembly=Syncfusion.Core.XForms"
```
Comment (karthikraja-arumugam) [inline MAUI/Themes/Keys.md:144]: Syncfusion.XForms => XF namespace?

File: MAUI/Themes/Keys.md [hunk 2]
```diff
@@ -107,3 +107,1809 @@ This page lists the keys for each control and the element to which it is mapped
...
+             ...>
+<Application.Resources>
+    <syncCore:SyncfusionThemeDictionary>
```
Comment (karthikraja-arumugam) [inline MAUI/Themes/Keys.md:147]: syncCore => namespace name mismatch

Comment (karthikraja-arumugam) [inline MAUI/Themes/Keys.md:147]: resource node is not correct use code from application

https://github.com/syncfusion-content/maui-docs/pull/1958

File: MAUI/Themes/Keys.md (hunk 1 above)
Comment (Shalini-Ashokan) [reply MAUI/Themes/Keys.md]: Updated

File: MAUI/Themes/Keys.md (hunk 2 above)
Comment (Shalini-Ashokan) [reply MAUI/Themes/Keys.md]: updated

Comment (Shalini-Ashokan) [reply MAUI/Themes/Keys.md]: Updated

File: MAUI/Themes/Keys.md [hunk 3]
```diff
@@ -107,3 +107,2093 @@ This page lists the keys for each control and the element to which it is mapped
...
+    </tr>
+
+	<tr>
+    <td>
+            SfImageEditorNormalToolbarIconColor       
+            <br/>
+            <br/>
+        </td>
+        <td>
+            Color of toolbar icon background in image editor.
```
Comment (karthikraja-arumugam) [inline MAUI/Themes/Keys.md:215]: Color of toolbar icon in image editor.

File: MAUI/Themes/Keys.md [hunk 4]
```diff
@@ -107,3 +107,2093 @@ This page lists the keys for each control and the element to which it is mapped
...
+        </td>
+    </tr>
+
+	<tr>
+    <td>
+            SfImageEditorDisabledToolbarIconColor        
+            <br/>
+            <br/>
+        </td>
+        <td>
```
Comment (karthikraja-arumugam) [inline MAUI/Themes/Keys.md:227]: Color of toolbar icon in image editor when it is disabled.

File: MAUI/Themes/Keys.md [hunk 5]
```diff
@@ -107,3 +107,2093 @@ This page lists the keys for each control and the element to which it is mapped
...
+    </tr>
+
+    <tr>
+    <td>
+            SfImageEditorNormalToolbarBackgroundColor     
+            <br/>
+            <br/>
+        </td>
+        <td>
+            Color of toolbar background in image editor.
```
Comment (karthikraja-arumugam) [inline MAUI/Themes/Keys.md:189]: Color of toolbar background.

Remove - **in image editor**, or **in Segment** from all the keys

File: MAUI/Themes/Keys.md [hunk 6]
```diff
@@ -107,3 +107,2093 @@ This page lists the keys for each control and the element to which it is mapped
...
+    </tr>
+
+	<tr>
+    <td>
+            SfImageEditorHoverToolbarIconBackground         
+            <br/>
+            <br/>
+        </td>
+        <td>
+            Color of toolbar icon background in image editor when it is hovered.
```
Comment (karthikraja-arumugam) [inline MAUI/Themes/Keys.md:241]: Color of toolbar icon when it is hovered.

File: MAUI/Themes/Keys.md [hunk 7]
```diff
@@ -107,3 +107,2093 @@ This page lists the keys for each control and the element to which it is mapped
...
+        </td>
+    </tr>
+
+	<tr>
+    <td>
+            SfImageEditorFocusedColorPaletteColor                 
+            <br/>
+            <br/>
+        </td>
+        <td>
```
Comment (karthikraja-arumugam) [inline MAUI/Themes/Keys.md:344]: Color of color palate icon when it is selected.

File: MAUI/Themes/Keys.md [hunk 8]
```diff
@@ -107,3 +107,2093 @@ This page lists the keys for each control and the element to which it is mapped
...
+    </tr>
+
+	<tr>
+    <td>
+            SfImageEditorSelectionStroke                  
+            <br/>
+            <br/>
+        </td>
+        <td>
+            Color of selection stroke in image editor.
```
Comment (karthikraja-arumugam) [inline MAUI/Themes/Keys.md:358]: Color of crop layer and annotations selection stroke.

File: MAUI/Themes/Keys.md [hunk 9]
```diff
@@ -107,3 +107,2093 @@ This page lists the keys for each control and the element to which it is mapped
...
+    </tr>
+
+	<tr>
+    <td>
+            SfImageEditorNormalSavePopupBackground                   
+            <br/>
+            <br/>
+        </td>
+        <td>
+            Color of save popup background in image editor.
```
Comment (karthikraja-arumugam) [inline MAUI/Themes/Keys.md:371]: Color of saved message popup background.

File: MAUI/Themes/Keys.md [hunk 10]
```diff
@@ -107,3 +107,2093 @@ This page lists the keys for each control and the element to which it is mapped
...
+    </tr>
+
+	<tr>
+    <td>
+            SfImageEditorNormalSavePopupTextColor                    
+            <br/>
+            <br/>
+        </td>
+        <td>
+            Color of save popup text in image editor.
```
Comment (karthikraja-arumugam) [inline MAUI/Themes/Keys.md:384]: Color of saved message popup text

File: MAUI/Themes/Keys.md [hunk 11]
```diff
@@ -107,3 +107,2093 @@ This page lists the keys for each control and the element to which it is mapped
...
+        </td>
+    </tr>
+
+	<tr>
+    <td>
+            SfImageEditorToolbarIconSeparatorColor                     
+            <br/>
+            <br/>
+        </td>
+        <td>
```
Comment (karthikraja-arumugam) [inline MAUI/Themes/Keys.md:396]: Color of tool bar icons divider line.

File: MAUI/Themes/Keys.md (hunk 3 above)
Comment (Shalini-Ashokan) [reply MAUI/Themes/Keys.md]: Updated

File: MAUI/Themes/Keys.md (hunk 4 above)
Comment (Shalini-Ashokan) [reply MAUI/Themes/Keys.md:227]: Updated

File: MAUI/Themes/Keys.md (hunk 6 above)
Comment (Shalini-Ashokan) [reply MAUI/Themes/Keys.md]: Updated

File: MAUI/Themes/Keys.md (hunk 7 above)
Comment (Shalini-Ashokan) [reply MAUI/Themes/Keys.md:344]: Updated

File: MAUI/Themes/Keys.md (hunk 8 above)
Comment (Shalini-Ashokan) [reply MAUI/Themes/Keys.md]: Updated

File: MAUI/Themes/Keys.md (hunk 9 above)
Comment (Shalini-Ashokan) [reply MAUI/Themes/Keys.md]: Updated

File: MAUI/Themes/Keys.md (hunk 10 above)
Comment (Shalini-Ashokan) [reply MAUI/Themes/Keys.md]: Updated

File: MAUI/Themes/Keys.md (hunk 11 above)
Comment (Shalini-Ashokan) [reply MAUI/Themes/Keys.md:396]: Updated

File: MAUI/Themes/Keys.md [hunk 12]
```diff
@@ -107,3 +107,2093 @@ This page lists the keys for each control and the element to which it is mapped
...
+    </tr>
+
+    <tr>
+    <td>
+            SfTreeMapSelectedItemStroke     
+            <br/>
+            <br/>
+        </td>
+        <td>
+            Color of treemap item when it is selected.
```
Comment (Karthickmani97) [inline MAUI/Themes/Keys.md:1999]: color of a treemap item's border when it is selected.

File: MAUI/Themes/Keys.md [hunk 13]
```diff
@@ -107,3 +107,2093 @@ This page lists the keys for each control and the element to which it is mapped
...
+    </tr>
+
+	<tr>
+    <td>
+            SfTreeMapItemHoverStroke      
+            <br/>
+            <br/>
+        </td>
+        <td>
+            Color of treemap item when it is hovered.
```
Comment (Karthickmani97) [inline MAUI/Themes/Keys.md:2012]: color of a treemap item's border when it is hovered.

File: MAUI/Themes/Keys.md [hunk 14]
```diff
@@ -107,3 +107,2093 @@ This page lists the keys for each control and the element to which it is mapped
...
+    </tr>
+
+	<tr>
+    <td>
+            SfTreeMapGroupItemNormalFontSize                   
+            <br/>
+            <br/>
+        </td>
+        <td>
+            Size of group item text.
```
Comment (Karthickmani97) [inline MAUI/Themes/Keys.md:2181]: Using `size` is fine, as it generally refers to both width and height.

File: MAUI/Themes/Keys.md [hunk 15]
```diff
@@ -107,3 +107,2093 @@ This page lists the keys for each control and the element to which it is mapped
...
+    </tr>
+
+	<tr>
+    <td>
+            SfSegmentedControlDisabledSegmentBackground       
+            <br/>
+            <br/>
+        </td>
+        <td>
+            Color of segment when it is disabled.
```
Comment (Karthickmani97) [inline MAUI/Themes/Keys.md:536]: Background Color of segment when it is disabled.

File: MAUI/Themes/Keys.md [hunk 16]
```diff
@@ -107,3 +107,2093 @@ This page lists the keys for each control and the element to which it is mapped
...
+    </tr>
+
+	<tr>
+    <td>
+            SfSegmentedControlDisabledSegmentTextColor        
+            <br/>
+            <br/>
+        </td>
+        <td>
+            Color of segment text when it is disabled.
```
Comment (Karthickmani97) [inline MAUI/Themes/Keys.md:549]: Color of segment item text when it is disabled.

File: MAUI/Themes/Keys.md [hunk 17]
```diff
@@ -107,3 +107,2093 @@ This page lists the keys for each control and the element to which it is mapped
...
+    </tr>
+
+	<tr>
+    <td>
+            SfSegmentedControlSelectionBackground         
+            <br/>
+            <br/>
+        </td>
+        <td>
+            Color of segment when it is selected.
```
Comment (Karthickmani97) [inline MAUI/Themes/Keys.md:562]: Background color of segment when it is selected.

File: MAUI/Themes/Keys.md [hunk 18]
```diff
@@ -107,3 +107,2093 @@ This page lists the keys for each control and the element to which it is mapped
...
+    </tr>
+
+	<tr>
+    <td>
+            SfSegmentedControlHoveredBackground            
+            <br/>
+            <br/>
+        </td>
+        <td>
+            Color of segment when it is hovered.
```
Comment (Karthickmani97) [inline MAUI/Themes/Keys.md:601]: Background color of segment when it is hovered.

File: MAUI/Themes/Keys.md [hunk 19]
```diff
@@ -107,3 +107,2093 @@ This page lists the keys for each control and the element to which it is mapped
...
+    </tr>
+
+	<tr>
+    <td>
+            SfSegmentedControlNormalStrokeThickness              
+            <br/>
+            <br/>
+        </td>
+        <td>
+            Size of stroke thickness in segemented control.
```
Comment (Karthickmani97) [inline MAUI/Themes/Keys.md:627]: can we use ? - Thickness of the stroke in the in segmented control.

Comment (Karthickmani97) [inline MAUI/Themes/Keys.md:627]: Spelling mistake - segemented ---> segmented

File: MAUI/Themes/Keys.md [hunk 20]
```diff
@@ -107,3 +107,2093 @@ This page lists the keys for each control and the element to which it is mapped
...
+    </tr>
+
+	<tr>
+    <td>
+            SfSegmentedControlNormalSegmentCornerRadius                
+            <br/>
+            <br/>
+        </td>
+        <td>
+            Size of segment corner radius.
```
Comment (Karthickmani97) [inline MAUI/Themes/Keys.md:653]: Please double-check – we need to specify the size.

File: MAUI/Themes/Keys.md [hunk 21]
```diff
@@ -107,3 +107,2093 @@ This page lists the keys for each control and the element to which it is mapped
...
+    </tr>
+
+	<tr>
+    <td>
+            SfTreeMapToolTipNormalBackground        
+            <br/>
+            <br/>
+        </td>
+        <td>
+            Color of tooltip in treemap.
```
Comment (Karthickmani97) [inline MAUI/Themes/Keys.md:2038]: Background color of tooltip in treemap.

File: MAUI/Themes/Keys.md [hunk 22]
```diff
@@ -107,3 +107,2093 @@ This page lists the keys for each control and the element to which it is mapped
...
+    </tr>
+
+	<tr>
+    <td>
+            SfTreeMapGroupItemNormalBackground           
+            <br/>
+            <br/>
+        </td>
+        <td>
+            Color of group item in treemap.
```
Comment (Karthickmani97) [inline MAUI/Themes/Keys.md:2077]: Background color of group item in treemap.

File: MAUI/Themes/Keys.md (hunk 12 above)
Comment (Shalini-Ashokan) [reply MAUI/Themes/Keys.md]: Updated

File: MAUI/Themes/Keys.md (hunk 13 above)
Comment (Shalini-Ashokan) [reply MAUI/Themes/Keys.md]: Updated

File: MAUI/Themes/Keys.md (hunk 15 above)
Comment (Shalini-Ashokan) [reply MAUI/Themes/Keys.md]: Updated

File: MAUI/Themes/Keys.md (hunk 16 above)
Comment (Shalini-Ashokan) [reply MAUI/Themes/Keys.md]: Updated

File: MAUI/Themes/Keys.md (hunk 17 above)
Comment (Shalini-Ashokan) [reply MAUI/Themes/Keys.md]: Updated

File: MAUI/Themes/Keys.md (hunk 18 above)
Comment (Shalini-Ashokan) [reply MAUI/Themes/Keys.md]: Updated

File: MAUI/Themes/Keys.md (hunk 19 above)
Comment (Shalini-Ashokan) [reply MAUI/Themes/Keys.md]: changed

File: MAUI/Themes/Keys.md [hunk 23]
```diff
@@ -107,3 +107,2093 @@ This page lists the keys for each control and the element to which it is mapped
...
+    </tr>
+
+	<tr>
+    <td>
+            SfSegmentedControlNormalCornerRadius               
+            <br/>
+            <br/>
+        </td>
+        <td>
+            Size of corner radius in segemented control.
```
[the remark this answers is not in the document]
Comment (Shalini-Ashokan) [reply MAUI/Themes/Keys.md]: Checked and changed

File: MAUI/Themes/Keys.md (hunk 21 above)
Comment (Shalini-Ashokan) [reply MAUI/Themes/Keys.md]: Updated

File: MAUI/Themes/Keys.md [hunk 24]
```diff
@@ -107,3 +107,2093 @@ This page lists the keys for each control and the element to which it is mapped
...
+    </tr>
+
+	<tr>
+    <td>
+            SfImageEditorTooltipBackground               
+            <br/>
+            <br/>
+        </td>
+        <td>
+            Color of tooltip.
```
Comment (karthikraja-arumugam) [inline MAUI/Themes/Keys.md:319]: Background Color of tooltip in Image editor toolbar

File: MAUI/Themes/Keys.md [hunk 25]
```diff
@@ -107,3 +107,2093 @@ This page lists the keys for each control and the element to which it is mapped
...
+    </tr>
+
+	<tr>
+    <td>
+            SfImageEditorTooltipTextColor                
+            <br/>
+            <br/>
+        </td>
+        <td>
+            Color of tooltip text.
```
Comment (karthikraja-arumugam) [inline MAUI/Themes/Keys.md:332]: Color of tooltip text in image editor

File: MAUI/Themes/Keys.md [hunk 26]
```diff
@@ -107,3 +107,2093 @@ This page lists the keys for each control and the element to which it is mapped
...
+    </tr>
+
+	<tr>
+    <td>
+            SfImageEditorTooltipTextFontSize                       
+            <br/>
+            <br/>
+        </td>
+        <td>
+            Size of tooltip text.
```
Comment (karthikraja-arumugam) [inline MAUI/Themes/Keys.md:423]: Font size on tooltip text in image editor

File: MAUI/Themes/Keys.md [hunk 27]
```diff
@@ -107,3 +107,2093 @@ This page lists the keys for each control and the element to which it is mapped
...
+    </tr>
+
+	<tr>
+    <td>
+            SfSegmentedControlSelectionTextColor          
+            <br/>
+            <br/>
+        </td>
+        <td>
+            Color of text when it is selected.
```
Comment (karthikraja-arumugam) [inline MAUI/Themes/Keys.md:575]: Color of segmented items text when it is selected.

File: MAUI/Themes/Keys.md [hunk 28]
```diff
@@ -107,3 +107,2093 @@ This page lists the keys for each control and the element to which it is mapped
...
+        </td>
+    </tr>
+
+	<tr>
+    <td>
+            SfSegmentedControlSelectionStroke           
+            <br/>
+            <br/>
+        </td>
+        <td>
```
Comment (karthikraja-arumugam) [inline MAUI/Themes/Keys.md:587]: Color of segmented item border when it is selected.

File: MAUI/Themes/Keys.md [hunk 29]
```diff
@@ -107,3 +107,2093 @@ This page lists the keys for each control and the element to which it is mapped
...
+    </tr>
+
+	<tr>
+    <td>
+            SfSegmentedControlNormalCornerRadius               
+            <br/>
+            <br/>
+        </td>
+        <td>
+            Size of corner radius in segmented control.
```
Comment (karthikraja-arumugam) [inline MAUI/Themes/Keys.md:640]: Corner radius of the segmented control.

File: MAUI/Themes/Keys.md (hunk 20 above)
Comment (karthikraja-arumugam) [reply MAUI/Themes/Keys.md:653]: Corner radius of the segment item in segmented control

File: MAUI/Themes/Keys.md [hunk 30]
```diff
@@ -107,3 +107,2093 @@ This page lists the keys for each control and the element to which it is mapped
...
+    </tr>
+
+	<tr>
+    <td>
+            SfSegmentedControlNormalFontSize                 
+            <br/>
+            <br/>
+        </td>
+        <td>
+            Size of text in segmented control.
```
Comment (karthikraja-arumugam) [inline MAUI/Themes/Keys.md:666]: Font size of the segment item in segmented control.

File: MAUI/Themes/Keys.md (hunk 25 above)
Comment (Shalini-Ashokan) [reply MAUI/Themes/Keys.md]: Updated

File: MAUI/Themes/Keys.md (hunk 26 above)
Comment (Shalini-Ashokan) [reply MAUI/Themes/Keys.md]: Updated

File: MAUI/Themes/Keys.md (hunk 27 above)
Comment (Shalini-Ashokan) [reply MAUI/Themes/Keys.md]: Updated

File: MAUI/Themes/Keys.md (hunk 28 above)
Comment (Shalini-Ashokan) [reply MAUI/Themes/Keys.md:587]: Updated

File: MAUI/Themes/Keys.md [hunk 31]
```diff
@@ -107,3 +107,2093 @@ This page lists the keys for each control and the element to which it is mapped
...
+    </tr>
+
+	<tr>
+    <td>
+            SfSegmentedControlKeyboardFocusStroke             
+            <br/>
+            <br/>
+        </td>
+        <td>
+            Color of border when it is keyboard pressed.
```
[the remark this answers is not in the document]
Comment (Shalini-Ashokan) [reply MAUI/Themes/Keys.md]: Updated

File: MAUI/Themes/Keys.md [hunk 32]
```diff
@@ -107,3 +107,2093 @@ This page lists the keys for each control and the element to which it is mapped
...
+    </tr>
+
+	<tr>
+    <td>
+            SfSegmentedControlNormalStrokeThickness              
+            <br/>
+            <br/>
+        </td>
+        <td>
+            Thickness of the stroke in the in segmented control.
```
[the remark this answers is not in the document]
Comment (Shalini-Ashokan) [reply MAUI/Themes/Keys.md]: Updated

File: MAUI/Themes/Keys.md (hunk 29 above)
Comment (Shalini-Ashokan) [reply MAUI/Themes/Keys.md]: Updated

File: MAUI/Themes/Keys.md (hunk 30 above)
Comment (Shalini-Ashokan) [reply MAUI/Themes/Keys.md]: Updated

File: MAUI/Themes/Keys.md (hunk 24 above)
Comment (Shalini-Ashokan) [reply MAUI/Themes/Keys.md]: Updated

File: MAUI/Themes/Keys.md (hunk 20 above)
Comment (Shalini-Ashokan) [reply MAUI/Themes/Keys.md]: We updated  Corner radius of segment in segmented control.

File: MAUI/Themes/Keys.md (hunk 19 above)
Comment (Shalini-Ashokan) [reply MAUI/Themes/Keys.md]: we used Thickness of the border stroke in the segmented control.

File: MAUI/Themes/Keys.md [hunk 33]
```diff
@@ -107,3 +107,2093 @@ This page lists the keys for each control and the element to which it is mapped
...
+        <td>
+            Text color of the dataform labels.    
+            <br/>
+            <br/>
+        </td>
+    </tr>
+
+	<tr>
+    <td>
+            SfDataFormDisabledLabelTextColor
```
Comment (karthikraja-arumugam) [inline MAUI/Themes/Keys.md:878]: remove this key

File: MAUI/Themes/Keys.md [hunk 34]
```diff
@@ -107,3 +107,2093 @@ This page lists the keys for each control and the element to which it is mapped
...
+    </tr>
+
+	<tr>
+    <td>
+            SfMapsShapeSubLayerNormalShapeFillColor                   
+            <br/>
+            <br/>
+        </td>
+        <td>
+            Color of sublayer in maps.
```
Comment (karthikraja-arumugam) [inline MAUI/Themes/Keys.md:1295]: add more details

File: MAUI/Themes/Keys.md [hunk 35]
```diff
@@ -107,3 +107,2093 @@ This page lists the keys for each control and the element to which it is mapped
...
+        </td>
+    </tr>
+    <tr>
+    <td>
+            SfMapsDataLabelTextFontSize                       
+            <br/>
+            <br/>
+        </td>
+        <td>
+            Font size on data label in maps.
```
Comment (karthikraja-arumugam) [inline MAUI/Themes/Keys.md:1469]: Font size of data label in maps.

File: MAUI/Themes/Keys.md [hunk 36]
```diff
@@ -107,3 +107,2093 @@ This page lists the keys for each control and the element to which it is mapped
...
+    </tr>
+
+	<tr>
+    <td>
+            SfTreeMapSelectedItemHoverStroke       
+            <br/>
+            <br/>
+        </td>
+        <td>
+            Color of selected item hovered in treemap.
```
Comment (karthikraja-arumugam) [inline MAUI/Themes/Keys.md:2025]: Color of a treemap selected item when it is hovered.

File: MAUI/Themes/Keys.md [hunk 37]
```diff
@@ -107,3 +107,2093 @@ This page lists the keys for each control and the element to which it is mapped
...
+    </tr>
+
+	<tr>
+    <td>
+            SfTreeMapSelectedItemStrokeWidth                
+            <br/>
+            <br/>
+        </td>
+        <td>
+            Size of selected item in treemap.
```
Comment (karthikraja-arumugam) [inline MAUI/Themes/Keys.md:2142]: Size of selected item border in treemap.

File: MAUI/Themes/Keys.md [hunk 38]
```diff
@@ -107,3 +107,2093 @@ This page lists the keys for each control and the element to which it is mapped
...
+        <td>
+            Color of scheduler when it is hovered.
+            <br/>
+            <br/>
+        </td>
+    </tr>
+
+    <tr>
+    <td>
+            SfSchedulerNormalBackground
```
Comment (karthikraja-arumugam) [inline MAUI/Themes/Keys.md:1552]: move background to first

File: MAUI/Themes/Keys.md [hunk 39]
```diff
@@ -107,3 +107,2093 @@ This page lists the keys for each control and the element to which it is mapped
...
+	</tr>
+    
+    <tr>
+    <td>
+            SfSchedulerHoverColor    
+            <br/>
+            <br/>
+        </td>
+        <td>
+            Color of scheduler when it is hovered.
```
Comment (karthikraja-arumugam) [inline MAUI/Themes/Keys.md:1544]: Color of scheduler cells when it is hovered

File: MAUI/Themes/Keys.md [hunk 40]
```diff
@@ -107,3 +107,2093 @@ This page lists the keys for each control and the element to which it is mapped
...
+    </tr>
+
+	<tr>
+    <td>
+            SfSchedulerNormalMoreAppointmentIndicatorColor       
+            <br/>
+            <br/>
+        </td>
+        <td>
+            Color of appointment indicator day, week and work week view.
```
Comment (karthikraja-arumugam) [inline MAUI/Themes/Keys.md:1583]: Color of the more appointment indicator icons.

File: MAUI/Themes/Keys.md [hunk 41]
```diff
@@ -107,3 +107,2093 @@ This page lists the keys for each control and the element to which it is mapped
...
+    </tr>
+
+	<tr>
+    <td>
+            SfSchedulerNormalSelectedAppointmentBackground          
+            <br/>
+            <br/>
+        </td>
+        <td>
+            Background Color of appointment layout while select week and work week view.
```
Comment (karthikraja-arumugam) [inline MAUI/Themes/Keys.md:1622]: Color of appointment border when it is selected.

File: MAUI/Themes/Keys.md [hunk 42]
```diff
@@ -107,3 +107,2093 @@ This page lists the keys for each control and the element to which it is mapped
...
+    </tr>
+
+	<tr>
+    <td>
+            SfSchedulerNormalTodayTextColor            
+            <br/>
+            <br/>
+        </td>
+        <td>
+            Current date text color of month cell.
```
Comment (karthikraja-arumugam) [inline MAUI/Themes/Keys.md:1648]: Color of today text in the scheduler

File: MAUI/Themes/Keys.md [hunk 43]
```diff
@@ -107,3 +107,2093 @@ This page lists the keys for each control and the element to which it is mapped
...
+    </tr>
+
+    <tr>
+    <td>
+            SfSchedulerNormalAllowedViewDropDownShadowBrush                       
+            <br/>
+            <br/>
+        </td>
+        <td>
+            Color of allowed view drop down shadow.
```
Comment (karthikraja-arumugam) [inline MAUI/Themes/Keys.md:1804]: Color of allowed view drop-down shadow.

File: MAUI/Themes/Keys.md [hunk 44]
```diff
@@ -107,3 +107,2093 @@ This page lists the keys for each control and the element to which it is mapped
...
+        </td>
+    </tr>
+
+	<tr>
+    <td>
+            SfSchedulerNormalViewHeaderDateTextColor                       
+            <br/>
+            <br/>
+        </td>
+        <td>
```
Comment (karthikraja-arumugam) [inline MAUI/Themes/Keys.md:1790]: Color of view header date text.

File: MAUI/Themes/Keys.md [hunk 45]
```diff
@@ -107,3 +107,2093 @@ This page lists the keys for each control and the element to which it is mapped
...
+    </tr>
+
+	<tr>
+    <td>
+            SfSchedulerNormalViewHeaderDayTextColor                      
+            <br/>
+            <br/>
+        </td>
+        <td>
+            Color of view header day text(Sunday,Monday,etc).
```
Comment (karthikraja-arumugam) [inline MAUI/Themes/Keys.md:1778]: Color of view header day text.

File: MAUI/Themes/Keys.md [hunk 46]
```diff
@@ -107,3 +107,2093 @@ This page lists the keys for each control and the element to which it is mapped
...
+    </tr>
+
+	<tr>
+    <td>
+            SfSchedulerNormalViewHeaderBackground                     
+            <br/>
+            <br/>
+        </td>
+        <td>
+            Background Color of view header layout.
```
Comment (karthikraja-arumugam) [inline MAUI/Themes/Keys.md:1765]: Background Color of view header.

File: MAUI/Themes/Keys.md [hunk 47]
```diff
@@ -107,3 +107,2093 @@ This page lists the keys for each control and the element to which it is mapped
...
+    </tr>
+
+	<tr>
+    <td>
+            SfSchedulerNormalHeaderBackground                   
+            <br/>
+            <br/>
+        </td>
+        <td>
+            Background Color of header layout in scheduler.
```
Comment (karthikraja-arumugam) [inline MAUI/Themes/Keys.md:1739]: Background Color of header in scheduler.

File: MAUI/Themes/Keys.md [hunk 48]
```diff
@@ -107,3 +107,2093 @@ This page lists the keys for each control and the element to which it is mapped
...
+        <td>
+            Background Color of appointment layout in day, week and work week view.    
+            <br/>
+            <br/>
+        </td>
+    </tr>
+
+	<tr>
+    <td>
+            SfSchedulerNormalAppointmentTextColor
```
Comment (karthikraja-arumugam) [inline MAUI/Themes/Keys.md:1695]: remove SfSchedulerNormalAppointmentTextColor

File: MAUI/Themes/Keys.md (hunk 33 above)
Comment (Shalini-Ashokan) [reply MAUI/Themes/Keys.md]: removed

File: MAUI/Themes/Keys.md (hunk 34 above)
Comment (Shalini-Ashokan) [reply MAUI/Themes/Keys.md]: Added

File: MAUI/Themes/Keys.md (hunk 35 above)
Comment (Shalini-Ashokan) [reply MAUI/Themes/Keys.md]: Updated

File: MAUI/Themes/Keys.md (hunk 37 above)
Comment (Shalini-Ashokan) [reply MAUI/Themes/Keys.md]: Updated

File: MAUI/Themes/Keys.md (hunk 38 above)
Comment (Shalini-Ashokan) [reply MAUI/Themes/Keys.md:1539]: Moved

File: MAUI/Themes/Keys.md (hunk 39 above)
Comment (Shalini-Ashokan) [reply MAUI/Themes/Keys.md]: Updated

File: MAUI/Themes/Keys.md (hunk 40 above)
Comment (Shalini-Ashokan) [reply MAUI/Themes/Keys.md]: Updated

File: MAUI/Themes/Keys.md (hunk 41 above)
Comment (Shalini-Ashokan) [reply MAUI/Themes/Keys.md]: Updated

File: MAUI/Themes/Keys.md [hunk 49]
```diff
@@ -107,3 +107,2093 @@ This page lists the keys for each control and the element to which it is mapped
...
+        </td>
+    </tr>
+    <tr>
+    <td>
+            SfSchedulerPressedAllowedViewMoreIconButtonBackground                       
+            <br/>
+            <br/>
+        </td>
+        <td>
+            Background Color of allowed view more icon button.
```
[the remark this answers is not in the document]
Comment (Shalini-Ashokan) [reply MAUI/Themes/Keys.md]: Updated

File: MAUI/Themes/Keys.md (hunk 43 above)
Comment (Shalini-Ashokan) [reply MAUI/Themes/Keys.md]: Updated

File: MAUI/Themes/Keys.md (hunk 44 above)
Comment (Shalini-Ashokan) [reply MAUI/Themes/Keys.md:1764]: Updated

File: MAUI/Themes/Keys.md (hunk 46 above)
Comment (Shalini-Ashokan) [reply MAUI/Themes/Keys.md]: Updated

File: MAUI/Themes/Keys.md [hunk 50]
```diff
@@ -107,3 +107,2093 @@ This page lists the keys for each control and the element to which it is mapped
...
+    </tr>
+
+	<tr>
+    <td>
+            SfSchedulerNormalTodayHighlightBrush         
+            <br/>
+            <br/>
+        </td>
+        <td>
+            Color of today hightlight.
```
[the remark this answers is not in the document]
Comment (Shalini-Ashokan) [reply MAUI/Themes/Keys.md]: Updated

**Review by florenceliliana — COMMENTED** [review]
The content provided is fine.

**Pull request closed without merging**